packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -111,13 +111,33 @@ function createInstanceMetadataServiceFactory(
           .getRegistrations()
           .map(feature => {
             if (feature.type === 'plugin') {
-              return { type: 'plugin', pluginId: feature.pluginId };
+              return Object.defineProperty(
+                {
+                  type: 'plugin',
+                  pluginId: feature.pluginId,
+                },
+                'toString',
+                {
+                  enumerable: false,
+                  configurable: false,
```

Comment:
Let's leave it configurable imo, little reason to lock that down here imo.

Reply:
Alright, updated!